navbar.js

```diff
@@ -0,0 +1,38 @@
+var categoriesSort = categories.sort((a, b) => {
+  return b.count - a.count;
+});
+
+var tmp = 0;
+
+var firstFiveCategory = categoriesSort.filter(c => {
+  tmp++;
+  return tmp <= 5;
+});
+var tmp1 = 0;
+var inne = categoriesSort.filter(c => {
```

Comment:
nigdy przenigdy nie nazywamy nic w kodzie po polsku, za to nam Kuba ucina rączki :D

Reply:
poza ucięciem rączek oczywiście jest to związane z pracą w zespole, nie zawsze nasz kolega z projektu mówi po polsku, nie zawsze nasz kod ląduje kiedyś w przyszłości u polskiego klienta czy innego zespołu developerów. zawsze po angielsku aby ułatwić pracę innym :)